runtime/jvmti/jvmtiObject.c

```diff
@@ -236,45 +236,44 @@ findMonitorThreads(J9VMThread *vmThread, J9JVMTIMonitorStats *pStats)
 
 		threadState &= ~(J9VMTHREAD_STATE_SUSPENDED | J9VMTHREAD_STATE_INTERRUPTED);
 		switch (threadState) {
-			case J9VMTHREAD_STATE_WAITING:
-			case J9VMTHREAD_STATE_WAITING_TIMED:
-				if (NULL == pStats->waiting) {
-					pStats->numWaiting += 1;
-				} else {
-					if (pStats->waitingCntr < pStats->numWaiting) {
-						pStats->waiting[pStats->waitingCntr] =
-								(jthread) vm->internalVMFunctions->j9jni_createLocalRef(jniEnv, target);
-						pStats->waitingCntr += 1;
-					}
+		case J9VMTHREAD_STATE_WAITING:
+		case J9VMTHREAD_STATE_WAITING_TIMED:
+			if (NULL == pStats->waiting) {
+				pStats->numWaiting += 1;
+			} else {
+				if (pStats->waitingCntr < pStats->numWaiting) {
+					pStats->waiting[pStats->waitingCntr] =
+							(jthread)vm->internalVMFunctions->j9jni_createLocalRef(jniEnv, target);
```

Comment:
If you're going to remove the space in casts like this, please fix them all (see lines 144, 153).

Reply:
Fixed.